### DISCUSSION TITLE
Error - ESLint: 7.32.0 ESLint couldn't find the config "standard" to extend from.

### DISCUSSION THREAD

**Discussion by daxdevd** [eslint/eslint#15718] [Q&A]
We have a CircleCI Pipeline. We have a older version of eslint i.e. 5.10.0 which was around three years ago installed and now client wants to upgrade node version to 16 and upgrade related packages.
I have modified package.json file and changes eslint version from  "eslint": "^5.10.0" to "eslint": "^7.10.0".
Now, while pushing changes on branch CircleCI test execution fails with the below error.:

```
ops! Something went wrong! :(

ESLint: 7.32.0

ESLint couldn't find the config "standard" to extend from. Please check that the name of the config is correct.

The config "standard" was referenced from the config file in "/home/xxx/xxx/.eslintrc.yml".

```

I have googled for this issue but could not find the solution for this.

**Comment by aladdin-add** [comment]
can you provide a repro? we won't be able to  help without the detailed info.

just a reminder: we had written migration guides for every major release, e.g. [migrating-to-7.0.0](https://eslint.org/docs/user-guide/migrating-to-7.0.0), you can upgrade eslint related deps, and follow the docs to see if it helps.

**Reply by daxdevd** [reply]
Thanks @aladdin-add for addressing my issue. 
I have also tried to use eslint ^8.11.0 but no luck.
Providing you with more info.
Below is my code structure:

> backend (directory)
>     ...
>     package.json
>     yarn.lock
> frontend (directory)
> .eslintrc.yml (file)
> docker-compose.yml (file)
> Dockerfile (file)

Now below is my backend/package.json file
```
{
  "scripts": {
    "dev-seed": "XXXXXX",
    "db-admin": "node node_modules/express-admin/app.js admin-config",
    "jest": "jest --env=node --no-cache",
    "lint": "eslint .",
    "migrate": "knex migrate:latest",
    "start": "node dist/server",
    "test": "yarn lint && yarn flow && yarn jest",
    "fix": "eslint . --fix",
    "wait-for-postgres": "while ! nc -z postgres 5432 2>/dev/null; do sleep 1; done",
    "watch": "node-dev src/dev"
  },
  "dependencies": {
    ...
  },
  "devDependencies": {
    "@babel/cli": "^7.1.5",
    "@babel/core": "^7.2.2",
    "@babel/eslint-parser": "^7.17.0",
    "@babel/plugin-proposal-class-properties": "^7.2.1",
    "@babel/plugin-proposal-json-strings": "^7.2.0",
    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
    "@babel/plugin-syntax-import-meta": "^7.2.0",
    "@babel/preset-env": "^7.2.0",
    "@babel/preset-flow": "^7.0.0",
    "@babel/register": "^7.0.0",
    "axios-mock-adapter": "^1.15.0",
    "babel-core": "^7.0.0-bridge",
    "babel-eslint": "^10.1.0",
    "babel-jest": "^27.5.1",
    "eslint": "^8.11.0",
    "eslint-config-standard": "^16.0.3",
    "eslint-config-standard-react": "^11.0.1",
    "eslint-plugin-import": "^2.25.4",
    "eslint-plugin-n": "^15.1.0",
    "eslint-plugin-node": "^11.1.0",
    "eslint-plugin-promise": "^6.0.0",
    "eslint-plugin-react": "^7.29.4",
    "flow-bin": "^0.89.0",
    "flow-typed": "^3.7.0",
    "jest": "^27.5.1",
    "node-dev": "^3.1.3",
    "prettier": "^1.15.2"
  },
  "jest": {
    "transform": {
      "^.+\\.js$": "babel-jest"
    }
  },
  "engines": {
    "node": ">=16"
  },
  "license": "UNLICENSED",
  "private": true
}

```
Now, I have .eslintrc.yml file on the root directory of the project and is as below
.eslintrc.yml
```
# backend/frontend common eslint rules

parser: 'babel-eslint'

extends:
  - standard
  - plugin:jest/recommended
  - prettier
  - prettier/standard

plugins:
  - jest
  - prettier

rules:
  no-console:
  - error
  - allow:
    - info
    - warn
    - error

  # Jest Rules
  jest/no-disabled-tests: off
  jest/no-focused-tests: off
  jest/no-identical-title: off
  jest/prefer-to-have-length: off
  jest/valid-expect: off

  prettier/prettier: error
```
------------------

While "yarn lint" command is being executed the posted error occurred and lint test goes fails.
I have checked with different versions and found that the same behavior occurs from eslint 6 to eslint 8 it is working fine on eslint 5.10.0

Let me know if you need any more info.
Thank you

**Reply by aladdin-add** [reply]
what's the output running `yarn lint --debug`?
